src/providers/auth-strategy.ts

```diff
@@ -12,8 +12,8 @@ import {
   AuthenticationMetadata,
 } from '@loopback/authentication';
 
-import {Strategy} from 'passport';
-import {BasicStrategy} from 'passport-http';
+import { Strategy } from 'passport';
+import { BasicStrategy } from 'passport-http';
```

Comment:
coding style - no spaces inside/around curly braces

please check and fix this issue in all files changed by this patch